ui/js/directives/treeherder/main.js

```diff
@@ -1,17 +1,3 @@
-treeherder.directive('ngRightClick', [
```

Comment:
Unused as of https://github.com/mozilla/treeherder/commit/191bc53f#diff-b246370db0e9db479d06d27c64740f80L6